### DISCUSSION TITLE
Hi everyone👋!!!

### DISCUSSION THREAD

**Discussion by alefram** [OpenMined/courses#109] [Introduce Yourself]
Hello everyone, my name is Alexis Fraudita, i am  from Venezuela and i happy to be in this great course. I am Electrical Engineer student right now doing my thesis about Reinforcement Learning in Robotics and learning a lot about software and ML.

 I am glad that these community exists because i have ADHD (Attention deficit hyperactivity disorder) and i always ask me  if i can create technology for help people like me to help with his attentions and other neurodevelopmental disorders like Autism, depression(that is a big problem in my country right now) etc using data. I always believe that machine learning, smart devices and good information flows can help this problems, from to understand why this things happen with data and build good solutions. 

So i hope to learn about AI privacy technologies to attach this problem and meet people who wants to create cool tech for help people neurodevelopmental disorders.🧠

**Comment by yemikifouly** [comment]
Hello Alex! 👋 Welcome to our community! 🙂 This is one of the most diverse and welcoming communities that I have been part myself and I have no doubt you will meet some great people who share your interest in creating tech for people with neurodevelopmental disorders. 

Consider browsing the posts in [Introduce Yourself](https://github.com/OpenMined/courses/discussions/categories/introduce-yourself). Maybe one of those who introduced themselves so far share your interest. Do not hesitate to reach out if you find someone who does. 🙂 

Once again, welcome and I hope you enjoy the course very much and remember to post any question you have. 🙂

**Reply by alefram** [reply]
Hii thanks for be our mentor, for sure i will see who wants learn with me in this area.
